RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -721,7 +718,7 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
                                                 itemMtime /= MS_IN_SEC;
                                             }
                                         } catch (ParseException ex) {
-                                            logger.log(Level.WARNING, "Failed to parse epoch time for installed program artifact.", ex); //NON-NLS
+                                            logger.log(Level.SEVERE, "Failed to parse epoch time for installed program artifact.", ex); //NON-NLS
```

Comment:
This seems like a malformed data exception, why is this logged as severe?

Reply:
They were originally SEVERE, I changed them to WARNING and I was asked by Richard to put them back the way they were.